packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -559,7 +559,7 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
 
         <oui-message
             data-type="warning"
-            data-ng-if="$ctrl.subnetGateways.length > 0 && !$ctrl.isGatewayLoading && $ctrl.selectedMode.name !== $ctrl.instanceModeEnum[1].mode"
+            data-ng-if="$ctrl.subnetGateways.length > 0 && !$ctrl.isGatewayLoading && $ctrl.selectedMode.name !== MODE_PUBLIC"
```

Comment:
Where this constant come from ?

Reply:
yup i missed a good way to add this constant in the template, i removed it